dataline-singer/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
 }
 
 jsonSchema2Pojo {
+    source = files("${sourceSets.main.output.resourcesDir}/singer_json")
```

Comment:
The problem here is that if this path is just `json` when we execute code in `dataline-workers`, the invocation of `prepareSchemas` that runs in `SingerConfigSchema`, only finds the resources from `dataline-config:models`, NOT `dataline-singer`. It seems like because they store their configs in the same relative path in their resources, one clobbers the other.

This seems to be an idiosyncrasy with resources that I don't fully understand. This unblocks us for now. I am going to read more carefully about java resources and hopefully come up with a less spooky solution. In the meanwhile, it seems like we need our config style packages to not store their json configs in paths with the same names.